creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -126,7 +131,56 @@ const retrieveClockValueForUserFromReplica = async (replica, wallet) => {
   return clockValue
 }
 
+/**
+ * Returns an object that can be returned from any state machine job to record a histogram metric being observed.
+ * Example: to call histogram.observe('response_time', { code: '200' }, 1000), you would call this function with:
+ * makeHistogramToRecord('response_time', 1000, 'code', '200')
+ * @param {string} metricName the key of the name of the metric from prometheus.constants
+ * @param {number} metricValue the value to observe
+ * @param {string} [metricLabelName] the optional name of the label
+ * @param {string} [metricLabelValue] the optional value of the label
+ */
+const makeHistogramToRecord = (
+  metricName,
+  metricValue,
+  metricLabelName = '',
+  metricLabelValue = ''
```

Comment:
a metric can have multiple labels, instead this should just be `metricLabels = {}`

Reply:
changed this and updated the validation to validate each label in the object